homeassistant/components/ws66i/models.py

```diff
@@ -0,0 +1,30 @@
+"""The ws66i integration models."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from pyws66i import WS66i
+
+from .coordinator import Ws66iDataUpdateCoordinator
+
+# A dataclass is basically a struct in C/C++
```

Comment:
We don't need to document Python features. They are documented in the Python docs.

Reply:
I was confused what dataclasses were. Once I made the connection to a data struct found in C, it made sense and I put it in there for my own benefit, but I'll remove the comment.